<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogAndCmdWindow</class>
 <widget class="QWidget" name="LogAndCmdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="logView">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="cmdEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>Saada</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
